src/containers/project-overview/index.tsx

```diff
@@ -1,20 +1,59 @@
+import { useEffect, useState } from 'react';
 import { Banner } from '../../components/banner';
 import { Footer } from '../../components/footer';
 import { ProjectsList } from '../../components/project/list';
+import { SupabaseClient } from '@supabase/supabase-js';
+import { Database } from '../../database.types';
+import { useNavigate } from 'react-router-dom';
+
+interface ProjectOverviewContainerProps {
+  supabase: SupabaseClient<Database>;
+}
+
+type Project = Database['public']['Tables']['project']['Row']
+
+export const ProjectOverview: React.FC<ProjectOverviewContainerProps> = ({ supabase }) => {
+
+  const navigate = useNavigate();
+
+  const [projects, setProjects] = useState<Project[]>([]);
+  const [errors, setErrors] = useState<String[]>([]);
+
+  async function handleSignOut() {
+
+    let { error } = await supabase.auth.getSession();
+    if (error)
```

Comment:
suggestion: instead of destructuring error from the response, why not we put it in a try catch block? so no need to also use a if else statement to resolve the error

Reply:
How would that work? Feel free to push up a commit that does it. I thought that it doesn't actually throw an error, it just returns an error object if there was an error, we can't use `try`/`catch`. It returns something like

```typescript
interface Response {
    data <null | Project>,
    error <null | PostgresError>
}
```